tests/packaging/test_package_contents.py

```diff
@@ -0,0 +1,20 @@
+import glob
+import sys
+import tarfile
+
+
+package = glob.glob('girder-[0-9].*.tar.gz')
+if not len(package):
+    print('Could not find package archive')
+    sys.exit(1)
+if len(package) > 1:
+    print('Error: multiple package versions found')
+    sys.exit(1)
+
+basename = package[0][:-7]  # remove '.tar.gz'
+
+with tarfile.open(package[0], 'r:gz') as tar:
+    headerMako = tar.getmember(basename + '/girder/mail_templates/_header.mako')
+    if headerMako.size <= 0:
+        print('Mail templates were not packaged properly')
+        sys.exit(1)
```

Comment:
While it is superseded by the new test in terms of functionality, it adds an additional layer that could make it easier to detect where an error is occurring (sdist step vs. install step) so I think we may as well keep it around since it's already written and runs in a tenth of a second.